### ISSUE TITLE
Continual Learning Experiment (Adaptation)

### ISSUE THREAD

**Issue by JonasFrey96** [leggedrobotics/wild_visual_navigation#135]
Description:
Domain Adaptation by switching between multiple scenes; Training Scenes in CL Setup; Train (F) - Evaluate (F,G,H)

Tasks:
- [ ] Modify training script to train with multiple data loaders in sequence (here check for warm start of optimizers)
- [ ] Modify training script to evaluate on multiple dataloaders
- [ ] Compute CL Metric 
- [ ] Create the matrix plot by comparing training sequentially without memory (G, F, H) vs with memory (G, GF, GFH) vs all at the same time (GFH) and the accuracy achieved when only trained on (G) (F) (H).
